protocol.txt

```diff
@@ -62,7 +65,43 @@ Table of Contents
 
 2. TCP Protocol
 
-                             <TO BE WRITTEN>
+    The TCP protocol will serve as the main protocol used by the client
+    and server before the start of a game. It will also serve the
+    purpose of handling anything non relevent to the game currently
+    being played. The sole exception to that last bit will be the
+    handling of in game messaging and server message broadcasting.
+
+    2.1. Command Syntax
+
+        TCP commands follow the following syntax :
+
+                <CMD> <SP> <HEADER> <SP> <ARGS> <CRLF>
+
+        Here is a definition of each part :
+
+        <CMD>    -> This is the command name and will always be fully
+                    capitalized.
+
+        <SP>     -> Ascii space character
+
+        <HEADER> -> Comma separated User UUID and Server UUID. Both are
+                    distributed by the server on connection.
+        
+        <ARGS>   -> Arguments for given command. These args are
+                    separated by the ',' ascii character.
+
+        <CRLF>   -> Carriage Return Line Feed character. Every message
+                    will end in this character combination denoted
+                    '\r\n' in ascii.
+
+    2.2. Command Response
+
+        All client sent commands will generate a response from the
```

Comment:
![image](https://user-images.githubusercontent.com/56591727/195980766-1eab1c30-4ec8-44b6-ba0e-c82eb5c3d25f.png)

It looks like a typo, Gimli